irods_specific_implementation/indexed_connection_pool_with_expiry.hpp

```diff
@@ -39,15 +39,19 @@ namespace irods {
 
             connection_handle(const std::string& _client_name)
             {
-                rodsEnv env;
-                _getRodsEnv(env);
+                const auto& env = irods::rest::configuration::instance().irods();
+                const char* host = env.at("host").get_ref<const std::string&>().data();
+                const auto  port = env.at("port").get<int>();
+                const char* zone = env.at("zone").get_ref<const std::string&>().data();
+                const char* rodsadmin_username = env.at("rodsadmin_username").get_ref<const std::string&>().data();
+                // TODO: Assumes client and proxy are in the same zone? And local?
```

Comment:
Can remove, but didn't know if we wanted to investigate this

Reply:
remove it for now - create new issue to investigate/consider